src/main/java/de/plushnikov/intellij/plugin/extension/LombokHighlightErrorFilter.java

```diff
@@ -1,35 +1,40 @@
 package de.plushnikov.intellij.plugin.extension;
 
+import java.util.regex.Pattern;
 import com.intellij.codeInsight.daemon.impl.HighlightInfo;
 import com.intellij.codeInsight.daemon.impl.HighlightInfoFilter;
 import com.intellij.lang.annotation.HighlightSeverity;
+import com.intellij.openapi.editor.colors.CodeInsightColors;
 import com.intellij.openapi.util.text.StringUtil;
 import com.intellij.psi.PsiFile;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import de.plushnikov.intellij.plugin.handler.BuilderHandler;
 import de.plushnikov.intellij.plugin.handler.EqualsAndHashCodeCallSuperHandler;
 import de.plushnikov.intellij.plugin.handler.LazyGetterHandler;
 import de.plushnikov.intellij.plugin.handler.OnXAnnotationHandler;
-import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
 
-import java.util.regex.Pattern;
 
 public class LombokHighlightErrorFilter implements HighlightInfoFilter {
   private static final Pattern UNINITIALIZED_MESSAGE = Pattern.compile("Variable '.+' might not have been initialized");
   private static final Pattern LOMBOK_ANYANNOTATIONREQUIRED = Pattern.compile("Incompatible types\\. Found: '__*', required: 'lombok.*AnyAnnotation\\[\\]'");
+  private static final Pattern VARIABLE_INITIALIZER_IS_REDUNDANT = Pattern.compile("Variable '.+' initializer '.+' is redundant");
   private static final String REDUNDANT_DEFAULT_PARAMETER_VALUE_ASSIGNMENT = "Redundant default parameter value assignment";
 
   @Override
   public boolean accept(@NotNull HighlightInfo highlightInfo, @Nullable PsiFile file) {
     if (null != file) {
-
+      String description = StringUtil.notNullize(highlightInfo.getDescription());
       if (HighlightSeverity.WARNING.equals(highlightInfo.getSeverity())) {
-        if (REDUNDANT_DEFAULT_PARAMETER_VALUE_ASSIGNMENT.equals(highlightInfo.getDescription())) {
+        if (REDUNDANT_DEFAULT_PARAMETER_VALUE_ASSIGNMENT.equals(description)) {
           return !EqualsAndHashCodeCallSuperHandler.isEqualsAndHashCodeCallSuperDefault(highlightInfo, file);
+        } else if (CodeInsightColors.NOT_USED_ELEMENT_ATTRIBUTES.equals(highlightInfo.type.getAttributesKey())) {
+          if (VARIABLE_INITIALIZER_IS_REDUNDANT.matcher(description).matches()) {
+            return !BuilderHandler.isDefaultBuilderValue(highlightInfo, file);
```

Comment:
You can try to write an inspection test, I think this warning is coming from DefUseInspection. Example: src/test/java/de/plushnikov/intellij/plugin/inspection/DataFlowInspectionTest.java

Reply:
Thanks for the example and review. I'll plane to cover it with a test. Also, I'm thinking about a small refactoring in this class. I'll push it in current PR.